### PR TITLE
update channels page

### PR DESCRIPTION
**Pull request by kevenboo**
Closes https://github.com/lncapital/torq/issues/167

- Added link to mempool
- Added link to amboss
- Added link to 1ml
- Converted gauge to percent
- PendingHtlc is now a count of  the pending HTLCs
- Added channel peer

Screen Shot 2022-10-21 at 5 19 40 PM
Screen Shot 2022-10-21 at 5 20 11 PM

### CODE REVIEW

File: internal/channels/handlers.go [hunk 1]
```diff
@@ -157,28 +174,31 @@ func getChannelListhandler(c *gin.Context, db *sqlx.DB) {
 
 		for _, channel := range r.Channels {
 			channelFee, err := client.GetChanInfo(context.Background(), &lnrpc.ChanInfoRequest{ChanId: channel.ChanId})
+			shortChannelId := ConvertLNDShortChannelID(channel.ChanId)
+			stringLNDShortChannelId := strconv.FormatUint(channel.ChanId, 10)
 			if err != nil {
 				server_errors.WrapLogAndSendServerError(c, err, "Channel info")
 				return
 			}
```
Comment (ohenrik) [inline internal/channels/handlers.go:182]: You are not immediately checking for error. Is there a reason for the channel ID conversions being in between the error source and the error check here?

File: web/src/features/table/cells/LinkCell.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import styles from "./cell.module.scss";
+// import { Link } from "react-router-dom";
```
Comment (ohenrik) [inline web/src/features/table/cells/LinkCell.tsx:2]: Delete commented out code

File: internal/channels/handlers.go [hunk 3]
```diff
@@ -192,23 +212,26 @@ func getChannelListhandler(c *gin.Context, db *sqlx.DB) {
...
 		return pendingHtlc
 	} else {
 		for _, htlc := range htlcs {
 			if htlc.ForwardingHtlcIndex == 0 {
 				continue
 			}
-			pendingHtlc += htlc.Amount
+			pendingHtlc++
 		}
 	}
```
[the remark this answers is not in the document]
Comment (ohenrik) [reply internal/channels/handlers.go:236]: I think we should keep all these columns as unselected by default as large tables can become slow in some browsers (looking at you Safari).

File: internal/channels/handlers.go (hunk 1 above)
Comment (kevenboo) [reply internal/channels/handlers.go]: No good reasons. When I moved them I did a bad copy past.

File: internal/channels/handlers.go [hunk 4]
```diff
@@ -149,6 +160,12 @@ func getChannelListhandler(c *gin.Context, db *sqlx.DB) {
 		defer conn.Close()
 
 		client := lnrpc.NewLightningClient(conn)
+
+		nodeInfo, _ := client.GetNodeInfo(context.Background(), &lnrpc.NodeInfoRequest{IncludeChannels: true, PubKey: *node.PubKey})
+		if err != nil {
+			server_errors.WrapLogAndSendServerError(c, err, "Node info")
+			return
+		}
```
[the remark this answers is not in the document]
Comment (kevenboo) [reply internal/channels/handlers.go]: oups my bad. Fixed

File: web/src/features/table/cells/LinkCell.tsx (hunk 2 above)
Comment (kevenboo) [reply web/src/features/table/cells/LinkCell.tsx]: Fixed

File: internal/channels/handlers.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (kevenboo) [reply internal/channels/handlers.go]: I added the new columns

**Pull request merged**